### ISSUE TITLE
2i AAE not reporting or repairing correctly [JIRA: RIAK-2293]

### ISSUE THREAD

**Issue by bsparrow435** [basho/riak_kv#1269]
Modified riak_test as well as manual testing shows 2i_aae is not repairing forcibly removed index entries although it is reporting as successful. Subsequent runs report the same entries but none are repaired.

Will add more information soon.

**Comment by awangc** [comment]
Any updates to this issue?

**Comment by bsparrow435** [comment]
@awangc yes, this work is currently blocked by our ongoing rework of AAE hashing to has only the vector clock and therefore eliminate the out-of-order metadata issue. This work is in progress but not yet complete. Once that is tested and merged, I can integrate my fix for the 2i AAE issue and make necessary changes based on that rework.
